heimrichhannot/contao-list-bundle
# add bs5 modal template and extend submission config

List config element type submission has been extended with following features:
 - Form template can now be selected
 - Multi-column fields for contextual mapping purpose

Testing:
- Set up OSD Raumgewinn project
- Go to newsroom, contact any person
- Contact person name should appear in submission inside contao backend
